wandb/board/ui/src/components/PanelLinePlot.js

```diff
@@ -106,7 +109,7 @@ class LinePlotPanel extends React.Component {
       return <p>This plot type is not supported on this page.</p>;
     }
     let data = this.props.data.history;
-    let keys = this.props.data.historyKeys;
+    let keys = this.props.data.historyKeys.filter(k => k !== 'examples');
```

Comment:
Is examples a special key?

Reply:
Ahhh, it was for a brief moment in time.  Will fix today.